build/config/msvc_base

```diff
@@ -0,0 +1,21 @@
+# Base for MSVC configs
+
+# Must be overridden
+CONFIG_NAME=msvc_base
+GENERATOR="GENERATOR_UNSET"
+BUILD_WITH_STATIC_RTLIB=BUILD_WITH_STATIC_RTLIB_UNSET
+
+# CMake
+SRC_DIR=.
+BUILD_DIR=build_$(CONFIG_NAME)
+INSTALL_DIR=install_$(CONFIG_NAME)
+
+FMILIB_CMAKE_CUSTOM_FLAGS=-DCMAKE_C_FLAGS= # N/A
+
+BUILD_TYPE=Release
+TEST_COMMAND=ctest --build-config $(BUILD_TYPE) --parallel $(NPROC) --output-on-failure
+
+# FMILIB_...
+GENERATE_DOXYGEN_DOC=0
+BUILD_TESTS=1
+TEST_LOCALE=1
```

Comment:
Extracted from msvc_md which was used as base for msvc_md_debug and msvc_mt.

A problem was that _mt inherited the GENERATOR from _md, which was not intuitive. They now independently depend on msvc_base.